src/common/types.ts

```diff
@@ -4,34 +4,35 @@ import CornellMajor from '../entities/CornellMajor';
 
 //** Represents a User */
 export interface SerializedUser {
-  clubs: SerializedClub[];
-  facebook: string | null;
-  firstName: string;
   googleID: string;
-  graduationYear: string | null;
-  hometown: string | null;
-  instagram: string | null;
-  interests: SerializedInterest[];
+  firstName: string;
   lastName: string;
   netID: string;
+  pronouns: string | null;
+  graduationYear: string | null;
   major: SerializedCornellMajor | null;
-  matches: SubSerializedMatching[];
+  hometown: string | null;
+  clubs: SerializedClub[];
+  interests: SerializedInterest[];
+  facebook: string | null;
+  instagram: string | null;
```

Comment:
not having it alphabetized makes it readable, but i worry a bit about keeping the order consistent across files in the future

Reply:
Yea, I was worrying about the future implications for this... but it seems like these attributes make much more sense logically related than not? When it was alphabetized, bio info like `firstName`, `lastName`, and `netID` were all over the place, which made parsing the response structure by eye fairly intricate. Not sure how big of a deal this is, but if y'all think alphabetized > logical relation - I'm down to change it.

Reply:
I would explore the option of alphabetizing but also categorizing by adding a comment or empty line between categories. Example:
```javascript
// Profile
firstName
lastName
netID
// Demographics
hometown
.
.
// Social Media
facebook
instagram
```